docs/src/python/user-guide/misc/visualization.py

```diff
@@ -0,0 +1,141 @@
+# --8<-- [start:setup]
+import base64
+
+import polars as pl
+
+url = "docs/data/iris.csv"
+
+# --8<-- [end:setup]
+
+# --8<-- [start:dataframe]
+
+iris_agg_df = (
+    pl.scan_csv(url)
+    .group_by("species")
+    .agg(
+        pl.col("petal_length").mean(),
+    )
+    .collect()
+)
+
+# --8<-- [end:dataframe]
+
+# --8<-- [start:hvplot_make_plot]
+import hvplot
+
+plot = iris_agg_df.plot.bar(
+    x="species",
+    y="petal_length",
+    width=650,
+)
+hvplot.save(plot, "docs/images/hvplot_bar.html")
+with open("docs/images/hvplot_bar.html", "r") as f:
+    chart_html = f.read()
+    print(f"{chart_html}")
+
+# --8<-- [end:hvplot_make_plot]
+"""
+# --8<-- [start:hvplot_show_plot]
+iris_agg_df.plot.bar(
+    x="species",
+    y="petal_length",
+    width=650,
+)
+# --8<-- [end:hvplot_show_plot]
+"""
+
+
+# --8<-- [start:matplotlib_make_plot]
+import matplotlib.pyplot as plt
+
+plt.bar(x=iris_agg_df["species"], height=iris_agg_df["petal_length"])
```

Comment:
what's `plt`? it's currently not visible to the user, they just see
```python
plt.bar(x=iris_agg_df["species"], height=iris_agg_df["petal_length"])
```

Reply:
I forgot to add those imports into the code displayed, I've done that now

Reply:
I can't see the import:

![image](https://github.com/pola-rs/polars/assets/33491632/828d5216-f95e-493e-96bb-3b756615c930)

Reply:
I don't know if I'm doing something wrong, so let's go over the steps:

1. I'm on your branch, on commit a80601775db7ac23258491432704908a88f3c8a3
2. I've done `make build` from the repo root
3. I've done `mkdocs serve`

I then opened localhost:8000 in a browser, and see:

![image](https://github.com/pola-rs/polars/assets/33491632/e2a50e0f-2fc5-41d5-9e26-e372503ecdad)


(without the `import matploblit.pyplot as plt` import)